src/test/java/uk/gov/companieshouse/api/strikeoffobjections/model/request/StrikeOffObjectionsRequestEntityTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.request;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.function.Supplier;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StrikeOffObjectionsRequestEntityTest {
+
+    private static final Supplier<LocalDateTime> MOCK_PRESENT = () -> LocalDateTime.of(2020, 6, 30, 10, 00);
+
+    @Test
+    public void test() {
```

Comment:
perhaps name this test something more descriptive, such as StrikeOffObjectionsRequestEntityBuilderTest